gapic/templates/README.rst.j2

```diff
@@ -48,3 +48,49 @@ Windows
     <your-env>\Scripts\activate
     <your-env>\Scripts\pip.exe install \path\to\library
 
+
+LOGGING
+-------
+
+This library has support to enable logging for debugging and monitoring purposes. Note that logs may contain sensitive information and care should be
```

Comment:
I'm not sure it's clear or accurate to say we support "enabling" logging. We log, in the sense we emit logging events. Users can set up handlers for those events, either explicitly via code or with the help of our environment-based config, which we describe below. I tried to clear this up in the suggested text.